stdlib/src/os/env.mojo

```diff
@@ -52,7 +52,23 @@ fn setenv(name: String, value: String, overwrite: Bool = True) -> Bool:
     return status == 0
 
 
-fn getenv(name: String, default: String = "") -> String:
+fn getenv(name: String) -> String:
+    """Returns the value of the given environment variable.
+
+    Constraints:
+      The function only works on macOS or Linux and returns an empty string
+      otherwise. The default is an empty string.
+
+    Args:
+      name: The name of the environment variable.
+
+    Returns:
+      The value of the environment variable.
+    """
+    return getenv(name, "")
+
+
+fn getenv(name: String, default: String) -> String:
```

Comment:
Should we change the signature to `fn getenv(name: String, default: String = "") -> String` and avoid introducing `fn getenv(name: String) -> String`?

Reply:
I cannot use default values because of this bug :( https://github.com/modularml/mojo/issues/2637 

When this is solved, I'll be able to remove those diffs